cves/kernel/CVE-2016-3135.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The vulnerability occurred during memory allocation where the size of the
+  struct being allocated was not properly checked. If the requested size of 
+  the allocation was too small, the memory heap could be corrupted due 
+  to integer overflow.
```

Comment:
The description is concise but technical, could be made easier to understand and written in simpler words. For example explain what memory heap and integer overflow is.

Reply:
Updated. Thanks!